Jint/Runtime/Environments/Binding.cs

```diff
@@ -14,5 +14,7 @@ public Binding(JsValue value, bool canBeDeleted, bool mutable)
         public JsValue Value;
         public readonly bool CanBeDeleted;
         public readonly bool Mutable;
+
+        public bool IsInitialized => !ReferenceEquals(Value, null);
```

Comment:
Need to add a mental note about this non-readonly-struct and methods causes defensive copies, we can revisit when things settle.